Simulator/components/GatherDataFrame.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component, PropTypes } from 'react';
+
+class GatherDataFrame extends Component {
+  render() {
+    return (
+      <iframe
+        style={{ display: 'none' }}
+        src={this.props.wdcUrl}
```

Comment:
The url bar is disabled in the data gather phase, but I considered keeping it enabled and only committing the change to the iframes url once the data gather phase is entered, that should be pretty easy to do if you think the UX would be better